include/tsid/tasks/task-actuation-equality.hpp

```diff
@@ -15,15 +15,56 @@
 // <http://www.gnu.org/licenses/>.
 //
 
-#ifndef __invdyn_tasks_fwd_hpp__
-#define __invdyn_tasks_fwd_hpp__
+#ifndef __invdyn_task_actuation_equality_hpp__
+#define __invdyn_task_actuation_equality_hpp__
+
+#include <tsid/tasks/task-actuation.hpp>
+#include <tsid/math/constraint-equality.hpp>
 
 namespace tsid
 {
   namespace tasks
   {
-    
+    class TaskActuationEquality : public TaskActuation
+    {
+    public:
+      EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+      typedef math::Index Index;
+      typedef math::Vector Vector;
+      typedef math::VectorXi VectorXi;
+      typedef math::ConstraintEquality ConstraintEquality;
+      typedef pinocchio::Data Data;
+
+      TaskActuationEquality(const std::string & name,
+                            RobotWrapper & robot);
+
+      int dim() const;
+
+      const ConstraintBase & compute(const double t,
+                                     ConstRefVector q,
+                                     ConstRefVector v,
+                                     Data & data);
+
+      const ConstraintBase & getConstraint() const;
+
+      void setReference(math::ConstRefVector ref); 
+      const Vector & getReference() const;
+
+      void setNormalizationVector(math::ConstRefVector norm);
+      const Vector & getNormalizationVector() const;
```

Comment:
I think "normalization vector" is not clear as a name. I'd suggest calling this "weight vector".

Reply:
Done